.github/ISSUE_TEMPLATE/release.md

```diff
@@ -7,17 +7,56 @@ assignees: ''
 
 ---
 
-Steps: 
+## Preparation on Thursday
 
+- [ ] Merge pull requests in the [Ready To Go] column
+- [ ] Merge [Transifex pull request]
 - [ ] Include translations: `tx pull --force`
-- [ ] [Draft new release]
-- [ ] Notify #instance-managers of user-facing changes.
-- [ ] Test: https://semaphoreci.com/openfoodfoundation/openfoodnetwork-2/branches/master <!-- replace the URL -->
-- [ ] Update translations if necessary
-- [ ] Publish and notify #global-community
-- [ ] Deploy and notify #instance-managers
+- [ ] [Draft new release]. Look at previous [releases] for inspiration.
+- [ ] Notify [#instance-managers] of user-facing changes.
+
+## Testing
+
+- [ ] [Find build] of the release commit and copy it below.
+- [ ] Move this issue to Test Ready and notify testers.
+- [ ] Test: :warning: link to the build of the release commit https://semaphoreci.com/openfoodfoundation/openfoodnetwork-2/branches/master
+
+## Finish on Tuesday
+
+- [ ] Update translations unless content has been removed from config/locales/en.yml between this release draft and current master.
+  <details><summary>Command line instructions</summary>
+  <pre>
+  git checkout master # same version as the release draft
+  git fetch upstream
+  git diff upstream master -- config/locales/en.yml
+  tx pull --force # if no changes or only additions in the locale
+  git checkout --detach # if we need to commit new translations
+  git commit -a -m "Update translations"
+  git tag vx.y.z # put the release number in here
+  git push upstream vx.y.z
+  </pre>
+  </details>
+- [ ] Publish and notify [#global-community]:
+  > The next release is ready: https://github.com/openfoodfoundation/openfoodnetwork/releases/latest
+- [ ] Deploy the new release to all managed instances.
+  <details><summary>Command line instructions</summary>
+  <pre>
+  cd ofn-install
+  git pull
+  (cd ../ofn-secrets && git pull)
```

Comment:
You shouldn't need to do anything with secrets whilst deploying...

Reply:
That's right! We should remove that. Or maybe we should add steps to provision if needed. What do you think?